packages/suite/src/views/settings/SettingsGeneral/Experimental.tsx

```diff
@@ -11,6 +11,8 @@ import { EXPERIMENTAL_FEATURES, ExperimentalFeature } from 'src/constants/suite/
 import { useDispatch, useSelector } from 'src/hooks/suite';
 import { selectIsDebugModeActive } from 'src/reducers/suite/suiteReducer';
 
+import { Link } from 'react-router-dom';
```

Comment:
This gives lint error (imports sorting)
:point_right: run `yarn lint:js --fix` to automatically sort them as per eslint :slightly_smiling_face:

Reply:
Solved in 919e03efc1165779a51769ce3298867753cfe1ba